jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -15196,6 +15196,9 @@ fi
     powerpc64)
       OPENJ9_CPU=ppc-64
       ;;
+    aarch64)
```

Comment:
This needs to be regenerated: it's missing the addition of `xr64` in the list of acceptable values of `OPENJ9_PLATFORM_CODE`.

Reply:
Sorry, but I don't understand what you mean.
Where is the list of acceptable values of `OPENJ9_PLATFORM_CODE`?

Reply:
That's exactly my point: it's not here (but it should be). The change to line 181 of `custom-hook.m4` is not reflected in this file.

Reply:
I added `xr64` to line 15872 of generated-configure.sh.

Reply:
Normally, you would just re-run `bash common/autoconf/autogen.sh`, but the result your way is fine.